csep/core/poisson_evaluations.py

```diff
@@ -4,14 +4,15 @@
 import numpy
 import scipy.stats
 import scipy.spatial
+import warnings
 
 from csep.models import EvaluationResult
 from csep.utils.stats import poisson_joint_log_likelihood_ndarray
 from csep.core.exceptions import CSEPCatalogException
-from csep.core.regions import QuadtreeGrid2D
 
 
-def paired_t_test(forecast, benchmark_forecast, observed_catalog, alpha=0.05, scale=False):
+def paired_t_test(forecast, benchmark_forecast, observed_catalog, alpha=0.05,
+                  scale=False):
```

Comment:
Try to split parameters across lines in a way that makes the lines more even. It is also ok to group them in an uneven way if it makes sense because of how they are used. See lines 171-173 for what a nice way would look like.

Reply:
I believe the plots module has a lot of formatting issues that we could review.  Also type-hinting would be highly appreciated. I did the mods you suggested, but I would prefer we pass this and the catalog PRs and then we could make a follow up PR fixing the formatting.

Reply:
That's true, we should do that. A follow-up PR can address all the formatting issues.